.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,95 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Buind And Run Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ '**' ]
+env:
+  VAULT_ADDR: https://clotho.broadinstitute.org:8200
+  VAULT_SERVICE_ACCOUNT_PATH: secret/dsde/terra/kernel/integration/yyu/crl_janitor/app-sa
+  VAULT_CLIENT_SERVICE_ACCOUNT_PATH: secret/dsde/terra/kernel/integration/yyu/crl_janitor/client-sa
```

Comment:
Is the plan to use the toolsalpha path instead? TODO?

Reply:
Changed to toolsalpha